pkg/driver/videotest/dummy.go

```diff
@@ -47,12 +47,10 @@ func (d *dummy) Close() error {
 }
 
 func (d *dummy) VideoRecord(p prop.Media) (video.Reader, error) {
-	yi := p.Width * p.Height
-	ci := yi / 2
+	if p.FrameRate == 0 {
```

Comment:
I think default value may depend on the device.
It might be better to have an interface to get default prop by the wrapper.

btw, https://github.com/pion/mediadevices/issues/61 is bit related.
videotest and screen can take any framerate, but the value the user specified is not passed to VideoRecord at now.

Reply:
Ah right. I think that I missed the main problem. I think the problem is not actuall about having the default value or not. But, the problem is how we handle non-discrete values, for example frame rate.

Reply:
I'll continue the conversation in https://github.com/pion/mediadevices/issues/61. I think the design decision for this can take a while.